Frameworks/UIKit/UIView.mm

```diff
@@ -409,6 +408,13 @@ - (UITouchPhase)_processPointerEvent:(WUXIPointerRoutedEventArgs*)pointerEventAr
             // Add this touch to the set of all current touches in the app
             [s_allTouches addObject:touchPoint.touch];
 
+            if (DEBUG_HIT_TESTING_LIGHT) {
+                TraceVerbose(TAG, L"Hit testing view %hs(0x%p).", object_getClassName(self), self);
+            }
+
+            // Update the hit-tested view. (Only one owner per UITouch lifetime)
+            touchPoint.touch->_view = [self _doHitTest:touchPoint.touch allTouches:s_allTouches];
```

Comment:
?


---
In reply to: [84356876](https://github.com/Microsoft/WinObjC/pull/1168#discussion_r84356876) [](ancestors = 84356876,84356602)